src/Input.d.ts

```diff
@@ -47,6 +47,7 @@ export interface InputProps
   type?: InputType;
   valid?: boolean;
   value?: string;
+  root?: HTMLElement;
```

Comment:
One comment is `root` typically means the outer component.

Could we rename this to `inner` or `element` or another suggestion?

Reply:
Ah, you're right. I think `inner` makes the most sense here, in that case.

I'll make that change now.